integration/tests/basic/rpc/test_rpc_base_calls.py

```diff
@@ -288,13 +324,15 @@ def test_eth_get_storage_at_eq_val(self, json_rpc_client):
         assert "error" not in response
         assert "result" in response
         assert new_data in web3.Web3.to_text(response["result"]), "wrong variable value"
+        EthGetStorageAt(**response)
 
-    @pytest.mark.mainnet
+    @pytest.mark.mainnet # for geth v1.14.7 the method eth_mining does not exist/is not available
```

Comment:
mark it please with only_neon mark and delete comment

Reply:
fixed ✅